Sources/ProjectDescription/RunAction.swift

```diff
@@ -26,7 +26,7 @@ public struct RunAction: Equatable, Codable {
     public let options: RunActionOptions
 
     /// List of diagnostics options to set to the action.
-    public let diagnosticsOptions: [SchemeDiagnosticsOption]
+    public let diagnosticsOptions: SchemeDiagnosticsOptions
```

Comment:
Would this class as a possible breaking change?

The syntax is still somewhat compatible for the common case

Reply:
restoring the old structure, we'll refactor to a struct in the next major as you suggested